ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -26,25 +26,29 @@
 # must include a trailing '/'.
 #
 - name: "pull runtime action images per manifest"
-  shell: "docker pull {{runtimes_registry | default()}}{{item.prefix}}/{{item.name}}:{{item.tag | default()}}"
+  shell: "docker pull {{runtimes_registry | default()}}{{inv_item.prefix}}/{{inv_item.name}}:{{inv_item.tag | default()}}"
```

Comment:
it will clash with "item" from controller deploy.yml
we need to include invoker deploy.yml anyway. either from separate plugin or from "when lean" section in controller deploy.yml